### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 8th of November 2022 17:30 CET / 12:30am EDT / 9:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#878] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 8th of November 2022  17:30 CET / 11:30am EDT / 9:30am PDT

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: [link](https://youtu.be/tICBMGqK9ZA)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## Agenda items
- OpenSSL CVE https://www.flatcar.org/releases/#release-3227.2.4
- uapi-group has a FOSDEM devroom - call for proposals! https://uapi-group.org/docs/conferences/2023-02-04__fosdem-devroom/
- LTS 2021 / LTS 2605 is end of life, update your nodes if you pinned the channel
- bug smashing Thursday 10th of Nov https://www.flowcode.com/page/flatcar

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by t-lo** [comment]
I'll do a brief recap of the OpenSSL CVEs, our security release process, and on back-porting patches vs. bumping versions.

**Comment by t-lo** [comment]
More ideas:
- uapi-group has a FOSDEM devroom - call for proposals!
- LTS 2021 / LTS 2605 is end of life, update your nodes if you pinned the channel

**Comment by t-lo** [comment]
Also, we might want to mention the bug smashing Thursday :smile:
